test/excludes/PostgresqlArrayTest.rb

```diff
@@ -5,7 +5,7 @@
 exclude :test_attribute_for_inspect_for_array_field_for_large_array, "Skipping until we can triage further. See https://github.com/cockroachdb/activerecord-cockroachdb-adapter/issues/48"
 exclude :test_column, "Skipping until we can triage further. See https://github.com/cockroachdb/activerecord-cockroachdb-adapter/issues/48"
 exclude :test_default, "Skipping until we can triage further. See https://github.com/cockroachdb/activerecord-cockroachdb-adapter/issues/48"
-exclude :test_insert_fixtures, "Skipping until we can triage further. See https://github.com/cockroachdb/activerecord-cockroachdb-adapter/issues/48"
+# exclude :test_insert_fixtures, "Skipping until we can triage further. See https://github.com/cockroachdb/activerecord-cockroachdb-adapter/issues/48"
```

Comment:
just checking -- did this one get commented on purpose?

Reply:
Sorry, no, I will remove this comment